src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -347,7 +366,7 @@ export class InputDatePicker
   /**
    * Fires when the component's value changes.
    */
-  @Event({ cancelable: false }) calciteInputDatePickerChange: EventEmitter<void>;
+  @Event({ cancelable: false }) calciteInputDatePickerChange: EventEmitter<DateRangeChange | void>;
```

Comment:
Let's roll this back. Per our conventions, event payloads should not include data that is accessible via a property.

Reply:
Good catch.  I got confused because I thought at first that we needed to include that event payload to keep from removing a deprecated feature, but since this is the new event that replaces the deprecated event, it doesn't make sense to keep emitting the payload.

Reply:
Cool, rolled that back.  I also added value checks in the test of what would be the EventTarget for the change event to make sure it looks right since the event detail is no longer being emitted.